apps/web/src/lib/components/TwitterCard.svelte

```diff
@@ -0,0 +1,167 @@
+<script lang="ts">
+	import { onMount } from 'svelte';
+
+	export let tweet: {
+		authorName: string;
+		authorHandle: string;
+		authorAvatar: string;
+		content: string;
+		date: string;
+		link: string;
+	};
+
+	let textElement: HTMLParagraphElement;
```

Comment:
```suggestion
  let textElement = $state<HTMLParagraphElement>();
```

Reply:
lol, this turned the whole file into "rune mode", which apparently is an entire can of Sveltey worms. I'll push this change off to a later time, since it seems to work fine - the twitter shit does not need to be dynamic or reactive.